mmdet/utils/misc.py

```diff
@@ -36,3 +40,38 @@ def find_latest_checkpoint(path, suffix='pth'):
             latest = count
             latest_path = checkpoint
     return latest_path
+
+
+def update_data_root(cfg, logger=None):
+    """Update data root according to env MMDET_DATASETS.
+
+    If set env MMDET_DATASETS, update cfg.data_root according to
+    MMDET_DATASETS. Otherwise, using cfg.data_root as default.
+
+    Args:
+        cfg (mmcv.Config): The model config need to modify
+        logger (logging.Logger | str | None): the way to print msg
+    """
+    if 'MMDET_DATASETS' not in os.environ:
+        print_log(
```

Comment:
I don't think it needs to be printed because nothing has changed. Avoid misleading users